scrapy/extensions/feedexport.py

```diff
@@ -131,18 +132,28 @@ def __init__(self, uri, access_key=None, secret_key=None):
     @classmethod
     def from_crawler(cls, crawler, uri):
         return cls(uri, crawler.settings['AWS_ACCESS_KEY_ID'],
-                   crawler.settings['AWS_SECRET_ACCESS_KEY'])
+                   crawler.settings['AWS_SECRET_ACCESS_KEY'],
+                   crawler.settings.get('FEED_STORAGE_S3_ACL'))
```

Comment:
I don't think you need `settings.get()` here because there is a default value of in `default_settings.py`

Reply:
Makes sense now that I introduced default values in the `default_settings.py` file.